packages/driver/src/cross-origin/events/cookies.ts

```diff
@@ -25,8 +25,8 @@ export const handleCrossOriginCookies = (Cypress: ICypress) => {
     // this event allows running a handler before stability is released.
     // this prevents subsequent commands from running until the cookies
     // are set via automation
```

Comment:
Should we move this comment down to where the event is used?

Reply:
Actually, it looks like it's already there and duplicated here, so this one could be removed?

Reply:
Removed in 66b323f74c